main.py

```diff
@@ -46,15 +46,23 @@
 def find_gst_processes():
     pids = []
     for child in psutil.process_iter():
-        if "Decky-Recorder" in " ".join(child.cmdline()):
-            pids.append(child.pid)
+        try:
+            if "Decky-Recorder" in " ".join(child.cmdline()):
+                pids.append(child.pid)
+        except:
```

Comment:
Please do not use bare except. At a minimum please do `except Exception`. Better way is to except on the specific error you expect to see. Best ways is to figure out what's causing the exception and handling that in the code. Is `child` None some times maybe? or child.cmdline is not returning a list and therefore the join fails?

Reply:
It's been a while so I don't remember off the top of my head what the exact error is, but it's to do with zombie processes. I'll have to debug this again to know for sure.

Reply:
Looks like I have some easily accessible logs currently on my steamdeck!
```
2023-11-07 21:07:42,527 - INFO - Traceback (most recent call last):
  File "/home/deck/homebrew/plugins/decky-recorder/main.py", line 118, in start_capturing
    await Plugin.clear_rogue_gst_processes(self)
  File "/home/deck/homebrew/plugins/decky-recorder/main.py", line 80, in clear_rogue_gst_processes
    gst_pids = find_gst_processes()
               ^^^^^^^^^^^^^^^^^^^^
  File "/home/deck/homebrew/plugins/decky-recorder/main.py", line 49, in find_gst_processes
    if "Decky-Recorder" in " ".join(child.cmdline()):
                                    ^^^^^^^^^^^^^^^
  File "/home/deck/homebrew/plugins/decky-recorder/backend/out/psutil/psutil/__init__.py", line 702, in cmdline
    return self._proc.cmdline()
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/deck/homebrew/plugins/decky-recorder/backend/out/psutil/psutil/_pslinux.py", line 1650, in wrapper
    return fun(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/deck/homebrew/plugins/decky-recorder/backend/out/psutil/psutil/_pslinux.py", line 1788, in cmdline
    self._raise_if_zombie()
  File "/home/deck/homebrew/plugins/decky-recorder/backend/out/psutil/psutil/_pslinux.py", line 1693, in _raise_if_zombie
    raise ZombieProcess(self.pid, self._name, self._ppid)
psutil.ZombieProcess: PID still exists but it's a zombie (pid=1275)
```

According to [psutils docs](https://psutil.readthedocs.io/en/latest/#psutil.NoSuchProcess), psutil.ZombieProcess extends from psutil.NoSuchProcess, so only NoSuchProcess needs to be caught.
> Raised by [Process](https://psutil.readthedocs.io/en/latest/#psutil.Process) class methods when no process with the given pid is found in the current process list, or when a process no longer exists. name is the name the process had before disappearing and gets set only if [Process.name()](https://psutil.readthedocs.io/en/latest/#psutil.Process.name) was previously called.
Judging from the description, I don't think any action needs to be taken and the loop can safely be continued. I'll update the code accordingly.